code/WorkInProgress/Cael_Aislinn/Rust/core_field.dm

```diff
@@ -33,15 +33,28 @@ Deuterium-tritium fusion: 4.5 x 10^7 K
 	var/datum/gas_mixture/held_plasma = new
 	var/particle_catchers[13]
 
+	var/temp = 0
+	var/temp_archived = 0
+	var/warning_point = 5000000
+	var/danger_point = 7500000
+	var/meltdown_point = 10000000
+	var/obj/item/device/radio/radio
```

Comment:
Wait actually I just checked, nevermind.

Apperently you do.
Thanks shitcoders.